### PR TITLE
Ft 2FA #187300150

### PR DESCRIPTION
**Pull request by manzidavidrw**
#### What does this PR do?
Two factor authentication (2FA) for seller specifically

#### Description of Task to be completed?

-provide OTP to  seller's registered email with speakeasy and nodemailer
-verify the provided OTP 

#### How should this be manually tested?

- use npm run dev to connect to the database

#### Any background context you want to provide?

-OTP is sent in cookies foe quick retrieval
-use api/users/login/verify to verify the OTP

#### What are the relevant pivotal tracker/Trello stories?

- id of the project
  #187300150

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,7 +144,12 @@
...
       "src/database/*",
       "src/config/*",
-      "src/utils/logger.ts"
+      "src/utils/logger.ts",
+      "src/utils/cloud.ts",
+      "src/middleware/fileUpload.ts",
+      "src/helper/excOperation.ts",
+      "src/helper/productQueries.ts",
+      "src/utils/sendEmail.ts"
     ]
```
Comment (fab-ryan) [inline package.json:153]: why this file removed

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Please rebase because it's seems you have not

File: package.json (hunk 1 above)
Comment (manzidavidrw) [reply package.json:153]: i removed this because it was bringing my test down and it requires internet to be tetsted

**Review by fab-ryan — CHANGES_REQUESTED** [review]
first rebase and squash the commits

**Pull request closed without merging**